src/vmspawn/vmspawn-util.c

```diff
@@ -236,3 +251,35 @@ int find_qemu_binary(char **ret_qemu_binary) {
 
         return find_executable(qemu_arch_specific, ret_qemu_binary);
 }
+
+int machine_cid(unsigned *ret_machine_cid) {
+        int r;
+        uint64_t cid;
```

Comment:
i am a bit confused now: is the CID of type uint64_t or unsigned? i.e. 64bit or 32bit?

Reply:
My understanding is that CID is of type unsigned, but the ioctl argument has type uint64_t.

Reply:
can you add a comment about that?

Reply:
Fixed in b0f65acd48.